product-web-app/views/register.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Register</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+</head>
+<body>
+    <header class="bg-success text-light">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+                    <h1>Register</h1>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="container mt-4">
+        <section id="form" class="bg-light py-5">
+            <div class="container">
+                {% if errormessage %}
+                    <div class="alert alert-danger" role="alert">
+                        {{ errormessage }}
+                    </div>
+                {% endif %}
+                <form action="/register" method="post">
+                    <div class="form-group">
+                        <label for="email">Email</label>
```

Comment:
Add an information about the fields requirements e.g. email must be in kainos domain, password must have special character etc.